README.md

```diff
@@ -76,33 +76,42 @@ $ adr-viewer --help
 
 Using adr-tools cli (adr):
 
-### set adr-tools directory for team in question:
-See the limitations of adr-tools notation above. You need to change `.adr-dir` to the team you want to work on.
-```
-$ echo "decisions/websre" > .adr-dir
-```
+To handle multiple teams we've made the table of contents generator work just on filenames.
```

Comment:
do you want to update the multiple teams note / wrapper comment above as you're starting to add that?